hphp/runtime/vm/jit/arg-group.h

```diff
@@ -188,6 +199,11 @@ struct ArgGroup {
     return *this;
   }
 
+  ArgGroup& indRet(Vreg base, Immed off) {
```

Comment:
Can you document here that on platforms which don't have indirect return argument registers, this behaves just like `addr`, and pushes a positional GP argument?